virtual-organizations/CMS.yaml

```diff
@@ -117,3 +117,4 @@ DataFederations:
     Namespaces:
       /store:
         - FQAN:/cms
+    AllowedCaches: []
```

Comment:
I'm not sure I understand this -- doesn't this prevent CMS from being in any automatically generated cache authfiles?

Reply:
@bbockelm ^^